src/MyLib.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DeriveGeneric, DeriveAnyClass, ForeignFunctionInterface #-}
+
+
+
+module MyLib (Scene(..), Camera(..), renderScene, setTargetImg, testBed, getMeanPixelValue, findTextureDifference) where
```

Comment:
Should we rename to something better - `CppFFI` or something more descriptive than `MyLib`